scripts/third_party_size_check.py

```diff
@@ -21,35 +21,54 @@
 
 import os
 import sys
+import yaml
 
 THIRD_PARTY_PATH = os.path.join(os.getcwd(), 'third_party')
 THIRD_PARTY_SIZE_LIMIT = 7000
 
 
-def _check_size_in_dir(dir_path):
+def _get_skip_files_list():
+    """This lists the directories which should be skipped."""
+    with open('./app.yaml', 'r') as app_yaml:
+        try:
+            app_yaml_dict = yaml.safe_load(app_yaml)
+        except yaml.YAMLError as exc:
+            print exc
+        skip_files = app_yaml_dict.get('skip_files')
+
+        skip_files = [os.getcwd() + '/' + skip_files_dir
+                      for skip_files_dir in skip_files]
+
+    return skip_files
+
+
+def _check_size_in_dir(skip_files, dir_path):
```

Comment:
Please swap the order of function parameters, `dir_path` should come before `skip_files_list`.

Reply:
Done Thanks!